src/deluge/model/sample/sample.cpp

```diff
@@ -1794,7 +1794,10 @@ void Sample::numReasonsDecreasedToZero(char const* errorCode) {
 
 		// LegsMechanical got, V4.0.0-beta2.
 		// https://forums.synthstrom.com/discussion/4106/v4-0-beta2-e078-crash-when-recording-audio-clip
+#if ALPHA_OR_BETA_VERSION
+		// this occurs very rarely and should only leak one cluster - better to not freeze in the official build
 		FREEZE_WITH_ERROR("E078");
+#endif
```

Comment:
this logic is already guarded

Reply:
Oh you're right, I missed that the whole function was wrapped in it